src/api-specification/contribution-controller/assembling-a-contribution.md

```diff
@@ -8,24 +10,56 @@
 
 **Method**: `POST`
 
-**Parameters**: —
+**Header**: `Authorization` — represented by a `string` that is the `accessToken` value that was returned either at the authorization, or the previous time the token [has been refreshed](refreshing-authentication-tokens.md).
 
-**Input/request structure**:
+#### Input/request structure
 
 ::: details Show
 
-```json5
+::: code-group Data structure
+
+```json5 [Structure]
 [
   {
-    id: 'string', //fraud identifier value
-    fraudType: 'string(enum)', //Could be one of the following: Wangiri, IRSF, StolenDevice, IPFraud, SMSA2P
-    origination: 'string', //two-digit country code (Alpha-2, ISO 3166)
-    destination: 'string', //two-digit country code (Alpha-2, ISO 3166)
-    expiryDate: 0 //integer($int32), Unix Epoch Time in seconds
+    "id": "idRange", //127.0.0.1 OR 127.0.0.1-127.0.0.2 OR +14155552671 OR +14155552671-+14155552672 OR 107615702016566
```

Comment:
I'd remove examples from the comment section and place them in the contribution format description page